app/views/static_pages/dashboard_steps/community_rules.md

```diff
@@ -55,24 +68,30 @@ We expect everyone to handle themselves professionally and treat others respectf
+-   **Provide context when sharing a resource.** When posting a link to an article or youtube video, include **why** you are sharing it. Otherwise it will be marked as spam and deleted.
 
-* **Post your finished projects in #creations-showcase.** This channel is a great collection of finished projects. Feedback happens in the #creations-discussions channel. Everyone is welcome to provide feedback on a project that you have completed.
+-   **Post your finished projects in #creations-showcase.** This channel is a great collection of finished projects. Feedback happens in the #creations-discussions channel. Everyone is welcome to provide feedback on a project that you have completed.
 
-* **Respect that people will have differences of opinion.** Every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer.
+-   **Respect that people will have differences of opinion.** Every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer.
 
-* **Remember the Human.** Behind every username there is a person with feelings, so be kind to everyone! If you wouldn't say it out loud, don't post it.
+-   **Be constructive, not destructive.** When providing feedback to another user, make sure you provide constructive feedback rather than feedback that is negative for the sake of being negative. Telling someone that their project is terrible or that they don't know how to write code doesn't provide them any actual help.
 
-* **Be gracious during misunderstandings.** When misunderstandings happen, choose to be gracious instead of argumentative. Give others the benefit of the doubt during text-based communication because you can not take into account body language, cultural differences, or other contextual cues.
+-   **Remember the Human.** Behind every username there is a person with feelings, so be kind to everyone! If you wouldn't say it out loud, or wouldn't want someone else saying it to you, don't post it.
+
+-   **Be gracious during misunderstandings.** When misunderstandings happen, choose to be gracious instead of argumentative. Give others the benefit of the doubt during text-based communication because you can not take into account body language, cultural differences, or other contextual cues.
 
 </div>
+
+#### 11. Be willing to change
+
+Not everything will result in a ban, but a lot of small things do add up. If you have been repeatedly told by a moderator to stop doing something, especially if your attitude and interactions within the community begin causing a toxic or unwelcoming environment, you may wind up being banned if you've proven to show zero effort in making changes. When you're informed about some form of behavior by a moderator, be open and willing to change your attitude to better fit the goals we wish to meet for our community.
```

Comment:
I do not think this needs to be a rule, because there is nothing to enforce. However, I like the jist of what you are saying and think it would fit better as a conclusion, because it is similar to the introduction above the rules.

```suggestion
Our moderation team enforces these rules to keep our community a friendly, safe and welcoming environment for everyone. If anyone repeatedly breaks rules, responds with negative/toxic attitudes, pushes the boundaries of our rules, or other “[help vampire](https://slash7.com/2006/12/22/vampires/)” behaviors, they will be banned.

If you are notified about your behavior by a moderator, we encourage you to be open and willing to change. Our rules are very similar to the expectations in professional workplaces, so this will better prepare you to interact professionally with your future teammates.
```